admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -979,6 +980,16 @@ var form = (function() {
       });
   }
 
+  function checkDOMChange() {
+    if($('.attribute_wholesale_price').length > 0 && $('.attribute_priceTE').length > 0) {
+      inputsTimeout = null;
+      $('.attribute_wholesale_price, .attribute_priceTE').each(function(){
+        $(this).val(priceCalculation.normalizePrice($(this).val()));
+      })
+    }else
+      inputsTimeout = setTimeout( checkDOMChange, 100 );
```

Comment:
> I think if you use the "DOMSubtreeModified" event instead of using "setTimeout", it will be better.
That would still be a workaround.

> Could you please give me some ideas for an alternative solution?
I think the form value is set server-side. I'd check if the form value is set correctly when the form is created.